src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/EventValidationServiceTest.java

```diff
@@ -380,7 +380,8 @@ void shouldValidateJurisdictionCodesWithUniqueCodes() {
     }
 
     @ParameterizedTest
-    @CsvSource({SUBMITTED_STATE + ",false", ACCEPTED_STATE + ",false", REJECTED_STATE + ",false", CLOSED_STATE + ",true"})
+    @CsvSource({SUBMITTED_STATE + ",false", ACCEPTED_STATE + ",false", REJECTED_STATE
+            + ",false", CLOSED_STATE + ",true"})
```

Comment:
Moving variables to new line is preferred to splitting a string of the same variable.
Have CLOSED_STATE begin the new line and move the REJECTED_STATE's "false" back a line